conans/client/remote_manager.py

```diff
@@ -64,7 +64,7 @@ def upload_recipe(self, conan_reference, remote, retry, retry_wait, ignore_delet
         ret, new_ref = self._call_remote(remote, "upload_recipe", conan_reference, the_files, retry, retry_wait,
                                          ignore_deleted_file, no_overwrite)
         duration = time.time() - t1
-        log_recipe_upload(new_ref, duration, the_files, remote)
+        log_recipe_upload(new_ref, duration, the_files, remote.name)
```

Comment:
Same here, it might make sense, in the same way it is receiving a ``ConanFileReference`` object, to receive a ``Remote`` object, then the logger internally will use whatever part of the object (.name) it wants to log.

Reply:
I don't agree here. the logger is always using the name, passing the whole object when it doesn't need it nor need to know about it is it a bad architecture.